dataverse-webapp/src/main/webapp/filesFragment.xhtml

```diff
@@ -269,67 +269,42 @@
-                    </p:commandLink>
-                    <!-- guest book or terms of use, etc. enabled - open "download popup" first: -->
-                    <p:commandLink rendered="#{datasetFilesTab.downloadPopupRequired}"
-                                    type="button" styleClass="btn btn-default btn-download"
-                                    disabled="#{false and datasetFilesTab.lockedFromDownload}"
-                                    action="#{datasetFilesTab.validateFilesForDownload(true, false)}"
+                                    action="#{datasetFilesTab.validateFilesForDownload(false)}"
                                     update="@form">
                         <span class="glyphicon glyphicon-download-alt"/> #{bundle.download}
                     </p:commandLink>
                 </div>
                 <div jsf:id="downloadButtonBlockTabular" class="btn-group"
-                     jsf:rendered="#{(!(empty datasetFilesTab.workingVersion.fileMetadatas) 
-                                    and datasetFilesTab.workingVersion.fileMetadatas.size() > 1) and datasetFilesTab.downloadButtonAvailable
+                     jsf:rendered="#{!(empty datasetFilesTab.workingVersion.fileMetadatas) 
+                                    and datasetFilesTab.downloadButtonAvailable
                                     and datasetFilesTab.isHasTabular()}">
                     <button type="button" class="btn btn-default btn-download dropdown-toggle" data-toggle="dropdown">
                             <span class="glyphicon glyphicon-download-alt"/> #{bundle.download} <span class="caret"/>
                         </button>
                         <ul jsf:id="downloadDropdownOptions" class="dropdown-menu multi-level pull-right text-left" role="menu">
-                            <li >
-                                <p:commandLink rendered="#{!(datasetFilesTab.downloadPopupRequired)}"
-                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
-                                                update="@form"
-                                                actionListener="#{datasetFilesTab.validateFilesForDownload(false, true)}">
-                                    #{bundle.downloadOriginal}
-                                </p:commandLink>
-                                <!-- guest book or terms of use, etc. enabled - open "download popup" first: -->
-                                <p:commandLink rendered="#{datasetFilesTab.downloadPopupRequired}"
-                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
-                                                action="#{datasetFilesTab.validateFilesForDownload(true, true)}"
+                            <li>
+                                <p:commandLink disabled="#{false and datasetFilesTab.lockedFromDownload}"
```

Comment:
If always `false` then why bother with second statement?

Reply:
Ok. I will remove it.